app/handlers/createHighlightVideo.py

```diff
@@ -83,6 +83,11 @@ def handler(identifier, email, ttc_threshold, vehicle_only):
             StatusHelper.set_status(identifier, Status.Type.HIGHLIGHT_VIDEO, Status.Flag.FAILURE)
             return (500, str(error_message))
 
+        near_misses = sorted(near_misses, key=(lambda x: x[4]))
+        num_near_misses_to_use = 5
```

Comment:
Sure, but we still should discuss a maximum value for this. I'd like to propose a semi-arbitrary 10?